packages/office-addin-dev-settings/src/commands.ts

```diff
@@ -389,21 +416,6 @@ function validateManifestId(manifest: ManifestInfo) {
   }
 }
 
-export async function webView(manifestPath: string, webViewString?: string) {
-  try {
-    const manifest = await readManifestFile(manifestPath);
-
-    validateManifestId(manifest);
-
-    if (webViewString) {
-      devSettings.setWebView(manifest.id!, parseWebViewType(webViewString));
-    } else {
-      const currentWebViewString = await devSettings.getWebView(manifest.id!);
-      currentWebViewString ? console.log("The webViewType is set to " + currentWebViewString + ".") : 
-      console.log("A specific webViewType override has not been selected.");
-    }
-
-  } catch (err) {
-    logErrorMessage(err);
-  }
+export async function webViewOverride(manifestPath: string, webViewString?: string) {
+  webViewString ? setWebViewType(manifestPath, webViewString) : getWebViewType(manifestPath);
```

Comment:
This looks good. You could name the parameter `webViewTypeString` for consistency.  As I mentioned before, this function could just be named webView().

Reply:
I've specifically set webViewType to be for the enums and webViewString to be for strings. By that logic, though, I could rename webViewType to webViewEnum 🤔

Reply:
Except that you typically shouldn't use "Enum" as part of the name for the enum. You would use what the enum describes which in this case is types of webviews.